src/styles/core/_changes.scss

```diff
@@ -19,7 +19,7 @@ $change-highlight-vertical-space-total: $change-highlight-vertical-space + $chan
 // Width of highlight endcaps only used in this file
 $change-total-coin-space: rem(5px);
 $change-coin-y-offset: 10px;
-$change-coin-y-offset-reduced: rem($change-highlight-vertical-space-total) + $change-total-coin-space/2;
+$change-coin-y-offset-reduced: rem($change-highlight-vertical-space-total) + $change-total-coin-space*0.5;
```

Comment:
![nice](https://media.giphy.com/media/yekL3gqpoRC92/giphy.gif)

Reply:
It easier to read than `math.div(` 🤣